.prettierrc

```
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "none",
```

As TS is a code update, I'd want to switch this to `all` as part of the PR - it saves updating 2 lines to add another entry where one one is needed (won't comment on any files as this can be run automatically) :-)

```suggestion
  "trailingComma": "all",
```